### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Deploy to Azure_Keyboard: Focus is not visible on "WebApp" combo-box

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3345]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier!

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 2.4.7

**Note: Same issue is observed in "IntelliJ" , "Dracula" and "High Contrast" themes**

**Repro Steps:**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and create new maven-archetype-webapp project
9) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
10) Under Project Explorer view, right-click your project, expand Azure, then click Deploy to Azure
11) Verify if the focus is visible on "WebApp" combo-box

**Actual Result:**
Focus is not visible on "WebApp" combo-box

**Expected Result:**
Focus should be visible on "WebApp" combo-box

**User Impact:**
Keyboard only users will not know if the focus is on the combo-box or not

**MAS Reference:**
[MAS 2.4.7](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})

[Bug_8_IntelliJ_Deploy to Azure_Focus Visible.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451310/Bug_8_IntelliJ_Deploy.to.Azure_Focus.Visible.pptx)
![Bug_8_IntelliJ_Edit Configuration_Focus Visible](https://user-images.githubusercontent.com/41108985/62206200-e047e400-b3ae-11e9-9849-320ca3da14e8.png)

**Comment by Flanker32** [comment]
@soumyatippapuram This issue has been fixed in #3541, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1250/Azure/), please help us verify it, thanks a lot!

**Comment by soumyatippapuram** [comment]
Verified this issue in the build mentioned above. This issue is fixed. Hence closing this. Added attachments for reference.
![Bug_3345_HighContrast](https://user-images.githubusercontent.com/41108985/65034479-c1a0ab00-d964-11e9-94ec-4a50d5e74881.png)
![Bug_3345_Dracula](https://user-images.githubusercontent.com/41108985/65034480-c2394180-d964-11e9-892d-d643bbc7b7b8.png)
![Bug_3345_IntelliJ](https://user-images.githubusercontent.com/41108985/65034481-c2394180-d964-11e9-99ca-100470381031.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749761
